index.js

```diff
@@ -64,6 +64,20 @@ async function smartContractInitialization() {
 
     const wallet = new Wallet(defaultPrivateKey, provider)
 
+    log(`Deploying SimpleTrackerRegistry contract from ${wallet.address}`)
+    const strDeploy = new ContractFactory(SimpleTrackerRegistry.abi, SimpleTrackerRegistry.bytecode, wallet)
+    const strDeployTx = await strDeploy.deploy(wallet.address, false, {gasLimit: 6000000} )
+    const str = await strDeployTx.deployed()
+    log(`SimpleTrackerRegistry deployed at ${str.address}`)
+    let tx = await str.createOrUpdateNode('0xb9e7cEBF7b03AE26458E32a059488386b05798e8', 'ws://10.200.10.1:30301')
+    await tx.wait()
+    tx = await str.createOrUpdateNode('0x0540A3e144cdD81F402e7772C76a5808B71d2d30', 'ws://10.200.10.1:30302')
+    await tx.wait()
+    tx = await str.createOrUpdateNode('0xf2C195bE194a2C91e93Eacb1d6d55a00552a85E2', 'ws://10.200.10.1:30303')
+    await tx.wait()
+    let nodes = await str.getNodes();
+    log(`TrackerRegistry nodes : ${JSON.stringify(nodes)}`)
+
```

Comment:
This doesn't make a difference. The streamr-docker-dev will still wait for the script to end.